### PR TITLE
460 service messaging sms otp

### PR DESCRIPTION
**Pull request by royadv**

### CODE REVIEW

File: migration/00001_init_tables.sql [hunk 1]
```diff
@@ -0,0 +1,31 @@
+-- +goose Up
+-- SQL in this section is executed when the migration is applied.
+
+CREATE TABLE "internals" (
```
Comment (ayannahcwinata) [inline migration/00001_init_tables.sql]: change to "clients"

File: migration/00001_init_tables.sql [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "id" bigserial,
+    "name" varchar(255) NOT NULL,
+    "key" varchar(255) NOT NULL,
+    "role" varchar(255) NOT NULL,
```
Comment (ayannahcwinata) [inline migration/00001_init_tables.sql:8]: client dont need role

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+ARG APPNAME="messaging"
+ARG ENV="dev"
+
+ADD . $GOPATH/src/"${APPNAME}"
+WORKDIR $GOPATH/src/"${APPNAME}"
+
+RUN apk add --update git gcc libc-dev;
+#  tzdata wget gcc libc-dev make openssl py-pip;
+RUN APK --no-chache add tzdata
```
Comment (ayannahcwinata) [inline Dockerfile]: masukin di line 9 aja

File: deploy/dev-config.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    password: asirapass
+    host: postgres_db
+    port: "5432"
+    maxlifetime: 30
+    idle_conns: 10
+    open_conns: 100
+    sslmode: disable
+    logmode: true
+    timezone: Asia/Jakarta
+  kafka:
```
Comment (ayannahcwinata) [inline deploy/dev-config.yaml]: remove kafka

File: handlers/messaging.go [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+		return returnInvalidResponse(http.StatusUnprocessableEntity, validate, "validation error")
+	}
+
+	//build otp request to partner
+	number := messaging.PhoneNumber
+	message := messaging.Message
+	var data = []map[string]string{
+		{"number": number, "message": message, "sendingdatetime": ""}, //time.Now().String()
+	}
+	conf := partner.Config{
```
Comment (ayannahcwinata) [inline handlers/messaging.go]: config should be set in init

**Review by ayannahcwinata — APPROVED** [review]

File: migration/00001_init_tables.sql (hunk 1 above)
Comment (royadv) [reply migration/00001_init_tables.sql]: ok : 463ef200aaad80ec8fdf68f8ec38fcc4035b0c66

File: Dockerfile [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+ARG APPNAME="messaging"
+ARG ENV="dev"
+
+ADD . $GOPATH/src/"${APPNAME}"
+WORKDIR $GOPATH/src/"${APPNAME}"
+
+RUN apk add --update git gcc libc-dev;
+#  tzdata wget gcc libc-dev make openssl py-pip;
+RUN apk --no-cache add tzdata
```
Comment (ayannahcwinata) [inline Dockerfile]: satuin di line 9

**Review by ayannahcwinata — CHANGES_REQUESTED** [review]

**Review by ayannahcwinata — APPROVED** [review]

**Pull request merged**